fdbserver/LogSystem.h

```diff
@@ -995,6 +995,11 @@ struct LogPushData : NonCopyable {
 		writtenLocations.clear();
 	}
 
+	void saveTags(std::set<Tag> s) {
```

Comment:
I suppose reference is needed here:
```suggestion
	void saveTags(std::set<Tag>& s) {
```

Reply:
+1